templates/account/components/oauth_tokens.html

```diff
@@ -1,6 +1,11 @@
 {% load i18n %}
 <section class="app-card">
-  <h2 class="pg-subtitle">{% translate "OAuth Access Tokens" %}</h2>
+  <div class="flex w-full justify-between">
+    <h2 class="pg-subtitle">{% translate "OAuth Access Tokens" %}</h2>
+    <a href="{% url 'oauth2_provider:application_home' %}" class="btn btn-outline btn-sm">
+      {% translate "Manage OAuth Applications" %}
```

Comment:
I'm between having a button here that leads to the list of applications, or just showing the list of applications above the Access token list. Somehow I like the current approach more, but suggestions welcome